detections/application/m365_copilot_jailbreak_attempts.yml

```diff
@@ -0,0 +1,61 @@
+name: M365 Copilot Jailbreak Attempts
+id: b05a4f25-e07d-436f-ab03-f954afa922c0
+version: 1
+date: 2025-09-24
+author: Rod Soto
+status: experimental
+type: TTP
+description: Detects M365 Copilot jailbreak attempts through prompt injection techniques including rule manipulation, system bypass commands, and AI impersonation requests that attempt to circumvent built-in safety controls.
+search: ' `m365_exported_ediscovery_prompt_logs`  
+| search Subject_Title="*pretend you are*" OR Subject_Title="*act as*" OR Subject_Title="*rules=*" OR Subject_Title="*ignore*" OR Subject_Title="*bypass*" OR Subject_Title="*override*"
+| eval jailbreak_score=case(
+   match(Subject_Title, "(?i)pretend you are.*amoral"), 4,
+   match(Subject_Title, "(?i)act as.*entities"), 3,
+   match(Subject_Title, "(?i)(ignore|bypass|override)"), 3,
+   match(Subject_Title, "(?i)rules\s*="), 4,
+   1=1, 1)
+| where jailbreak_score >= 2
+| table _time, Sender, Subject_Title, jailbreak_score, Workload, Size
```

Comment:
Use this `| eval user = Sender` and output user everywhere in the yaml

Reply:
done